RustyHog: improve description and file_path (#11433)
DefectDojo/django-DefectDojo: docs/content/en/open_source/upgrading/2.42.md

weight: -20241202
description: No special instructions.
---

**Hash Code changes**

> The Rusty Hog parser has been [updated](https://github.com/DefectDojo/django-DefectDojo/pull/11433) to populate more fields. Some of these fields are part of the hash code calculation. To recalculate the hash code please execute the following command:

Reviewer: ```suggestion
A few parsers have been updated to populate more fields. Some of these fields are part of the hash code calculation. To recalculate the hash code please execute the following command:
```
